arborist/action.go

```diff
@@ -0,0 +1,102 @@
+package arborist
+
+import (
+	"errors"
+)
+
+// The wildcard "*" in a permission or action field denotes that the field
+// should validate against any input.
+const WILDCARD string = "*"
```

Comment:
would be nice if we have a 'not' concept. So we can specify a role that can do everything except for some super admin stuff